backend/src/main/java/com/hyooun/wilson/review/domain/Review.java

```diff
@@ -57,52 +46,24 @@ public static Review withId(ReviewId id,
 		Project.ProjectId projectId,
```

Comment:
Review 클래스의 생성자 withId 메서드 시그니처가 변경되었습니다.

```suggestion
public static Review withId(ReviewId id, Project.ProjectId projectId, String title, String body, List<Change> changes, ReviewResult reviewResult) {
```